amazon_msk/datadog_checks/amazon_msk/check.py

```diff
@@ -112,11 +112,14 @@ def configure_transformer_with_metric_label(self, legacy_name, new_name, label_n
         def transform(metric, sample_data, runtime_data):
             for sample, tags, hostname in sample_data:
                 method(legacy_name, sample.value, tags=tags, hostname=hostname)
+                self.count(legacy_name + ".count", sample.value, tags=tags, hostname=hostname)
 
                 tag = sample.labels.pop(label_name)
                 tags.remove('{}:{}'.format(label_name, tag))
 
-                method('{}.{}'.format(new_name, tag), sample.value, tags=tags, hostname=hostname)
+                new_name_to_submit = '{}.{}'.format(new_name, tag)
+                method(new_name_to_submit, sample.value, tags=tags, hostname=hostname)
+                self.count(new_name_to_submit + ".count", sample.value, tags=tags, hostname=hostname)
```

Comment:
To make sure I understand the two `self.count`s, the first `self.count` is for `RequestMetrics.Count` and `BrokerTopicMetrics.Count` while the second `self.count` is for metrics from label id?